Core/LocalAdmin.cs

```diff
@@ -183,45 +184,53 @@ internal async Task Start(string[] args)
 
             if (DataJson!.EulaAccepted == null)
             {
-                ConsoleUtil.WriteLine($"Welcome to LocalAdmin version {VersionString}!", ConsoleColor.Cyan);
-                ConsoleUtil.WriteLine("Before starting please read and accept the SCP:SL EULA.", ConsoleColor.Cyan);
-                ConsoleUtil.WriteLine("You can find it on the following website: https://link.scpslgame.com/eula", ConsoleColor.Cyan);
-                ConsoleUtil.WriteLine("", ConsoleColor.Cyan);
-                ConsoleUtil.WriteLine("Do you accept the EULA? [yes/no]", ConsoleColor.Cyan);
-
-                ReadInput((input) =>
-                    {
-                        if (input == null)
-                            return false;
+                if (args.Contains("--acceptEULA", StringComparer.Ordinal) ||
+                    Environment.GetEnvironmentVariable("SLLA_EULA")?.ToUpperInvariant() is "1" or "TRUE")
+                {
+                    DataJson!.EulaAccepted = DateTime.UtcNow;
+                    _autoEula = true;
```

Comment:
1. Actually the variable `_autoEula` is not needed, because you can print the message here.
2. Add `await SaveJsonOrTerminate();` here.

Reply:
> Actually the variable _autoEula is not needed, because you can print the message here.
It will be cleared when the Menu method is called, won't it? 
```cs
private static void Menu()
{
        ConsoleUtil.Clear();
        ConsoleUtil.WriteLine($"SCP: Secret Laboratory - LocalAdmin v. {VersionString}", ConsoleColor.Cyan);
        ConsoleUtil.WriteLine(string.Empty, ConsoleColor.Cyan);
        <...>
 ```

Reply:
Right, there is the clear command.